packages/core/src/capabilities/NodeInfo.ts

```diff
@@ -124,32 +124,47 @@ export function parseCCList(payload: Buffer): {
 	};
 	let offset = 0;
 	let isAfterMark = false;
-	while (offset < payload.length) {
+	let listEnd = payload.length;
+	if (isLongRange) {
+		validatePayload(payload.length >= offset + 1);
+		const listLength = payload[offset++];
+		listEnd = offset + listLength;
+		validatePayload(payload.length >= listEnd);
+	}
+	while (offset < listEnd) {
 		// Read either the normal or extended ccId
 		const { ccId: cc, bytesRead } = parseCCId(payload, offset);
 		offset += bytesRead;
 		// CCs before the support/control mark are supported
 		// CCs after the support/control mark are controlled
+		// BUGBUG: does the "mark" and support/control convention apply to isLongRange?
```

Comment:
I think the answer to this was "yes, this code already supports multi-byte values"